### ISSUE TITLE
Express js vue-resource.common.js:641 var ie = document.documentMode;

### ISSUE THREAD

**Issue by marcoapferegrino** [pagekit/vue-resource#527]
I am using express js, with whit packages 
```
"vue": "^2.1.4",
"vue-resource": "^1.0.3"
```

With this app.js 

```
Vue = require('vue'),
VueResource = require('vue-resource'),

Vue.use(VueResource);
```
But I get this error

vue-resource.common.js:641
var ie = document.documentMode;
                 ^

ReferenceError: document is not defined
    at Object.<anonymous> (/Users/Marco/Documents/NodeProjects/inmersys/node_modules/vue-resource/dist/vue-resource.common.js:641:18)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/Marco/Documents/NodeProjects/inmersys/app/app.js:9:15)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)

What does mean ??

**Comment by OmarKRostom** [comment]
Did you find a solution, am facing the same exact problem ...

**Comment by marcoapferegrino** [comment]
No i could not yet :(

**Comment by ahuezo** [comment]
Same problem. Any solution?

**Comment by nicolegithub** [comment]
the same!

**Comment by steffans** [comment]
Fixed in v1.1.2
